scripts/extract_mecas.sh

```diff
@@ -84,7 +84,7 @@ for file in $INCOMING_DIR/*; do
     done
 
     echo "Introduce PDF, if available"
-    aws s3 sync ${PDF_S3_BUCKET}/${doi} ${outputDir} --exclude "*" --include "*.pdf" --delete
+    aws s3 sync ${PDF_S3_BUCKET}/${doi} ${outputDir}
```

Comment:
I'm not sure how this wasn't spotted sooner but this will mean that files were being deleted.